src/Effects.js

```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.music_wave = {
+      wave: [],
+      init: function () {
+        let wave = {
+          inc: 360/15,
+          heightFactor: 1,
+          yHalf: 300,
```

Comment:
The name of this variable, `yHalf`, makes it seem as though it represents half of the total height of the screen, which I believe would have a value of 200. Can we update this variable name so it doesn't imply that it is half the height?

Reply:
I realize now that the variable was to mark the y location of the center of the wave. I updated the name to reflect this.